examples/01_dirty_categories.py

```diff
@@ -261,6 +259,7 @@
 # Let's perform a cross-validation to see how well this model predicts:
 
 from sklearn.model_selection import cross_val_score
+import numpy as np
```

Comment:
Hi, thanks for this PR! 
Here, I believe that you need to import numpy earlier (for line 182).

Reply:
Right, I see 
Should I close pull request and create new one or somehow edit this one?

Reply:
No, you can edit this one and iterate as many times as you want!
You can do this by making [new commits and then pushing them](https://www.earthdatascience.org/workshops/intro-version-control-git/basic-git-commands/) to github.

Reply:
Hi @kazmierczakP, you only have to `git commit ...` and `git push` on your laptop, and then the commits will land on this PR :)